makefile.shade

```diff
@@ -0,0 +1,8 @@
+
+var VERSION='0.1'
+var FULL_VERSION='0.1'
+use-standard-lifecycle
+k-standard-goals
+
+#restore-nuget-packages target='initialize'
+    exec program='${Path.Combine(Directory.GetCurrentDirectory(), ".build", "nuget.exe")}' commandline='restore Mvc.sln'
```

Comment:
The equivalent code in Razor was to support building the vsix, so I don't think this is needed here. If you're trying to "select" the sln, you need to use `dotnet restore`